server/routes/api/articles.js

```diff
@@ -0,0 +1,38 @@
+import { Router } from 'express';
+import auth from '../../utils/middleware/AuthMiddleware';
+import validate from '../../utils/middleware/validator/articles';
+import ArticleController from '../../controllers/ArticleController';
+
+
+const router = Router();
+
+router.post(
+  '/articles',
+  validate.createArticle,
+  auth.authenticateUser,
```

Comment:
You need to authenticate before you validate. Because there is no need to validate an input for a user that is not logged in

Reply:
Done